ci-operator/step-registry/image-based-install/ztp/preinstall-host/image-based-install-ztp-preinstall-host-commands.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install ztp preinstall host command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+export IBI_BHC=\$(cat \${EXTRA_BAREMETALHOSTS_FILE} | jq '.[0].driver_info.address')
```

Comment:
```suggestion
export IBI_BHC=\$('cat ${EXTRA_BAREMETALHOSTS_FILE} | jq '.nodes.nodes[0].driver_info.address')
```

Reply:
I need the "\" before the "$" because the envs are set in the source files on the remote server.
And that file doesn't have the "nodes.node" part (this file was created from extracting that part from another file).